client/src/views/AboutUS.js

```diff
@@ -372,10 +372,9 @@ const AboutUS = () => {
                                     <blockquote className="text-left text-gray-500 text-sm md:text-base mb-6">
                                         <p>{technology.description}</p>
                                     </blockquote>
-                                    <a href={technology.link} target="_blank" rel="noopener noreferrer" 
-                                    className="text-transparent bg-gradient-to-r from-cyan-500 to-blue-500 bg-clip-text text-left w-full font-medium hover:text-indigo-700 mb-4 transition-colors delay-150 duration-300 ease-in-out">
-                                        Learn more <i className="fa-solid fa-caret-right"></i>
-                                    </a>
+                                    <h1 className="place-self-start text-transparent bg-gradient-to-r from-cyan-500 to-blue-500 bg-clip-text font-medium hover:text-indigo-700 mb-4 transition-colors delay-150 duration-300 ease-in-out">
```

Comment:
Instead of `text-left w-full`, I replace it with `place-self-start` and solves the issue.
and the photo below shows how its done.
hovering in a empty space should do nothing as expected
![image](https://user-images.githubusercontent.com/95494414/188169799-892a21e5-a0ad-4e24-9ba7-da607584afc1.png)

and when it hovers to the text
![image](https://user-images.githubusercontent.com/95494414/188169708-e5962455-df5c-4d42-b187-078a6c02c373.png)